babel.config.js

```diff
@@ -31,24 +31,32 @@ const argv = minimist(process.argv.slice(2));
 
 // eslint-disable-next-line amphtml-internal/no-module-exports
 module.exports = function(api) {
+  const isDist = argv._.includes('dist');
+  const {esm} = argv;
+  const es5Target = {
```

Comment:
Let's use the names `module` and `nomodule` for targetting to be consistent with their purpose.

Reply:
done